src/dataflow/src/sink/kafka.rs

```diff
@@ -581,9 +581,10 @@ where
 
                         match result {
                             Ok(()) => {
+                                // sanity check for the continuous updating
+                                // of the write frontier below
                                 assert!(write_frontier.borrow().less_equal(ts));
```

Comment:
is `less_than` technically correct here now?

Reply:
`less_than` would not work, I added a description here that mirrors what I wrote in the commit message. Do you think that's a problem?

Reply:
@ruchirK gentle ping. :sweat_smile:

Reply:
Sorry I hit approve but forgot to respond here -- the rationale in the comment makes sense to me!